apps/backend/.env.example

```
@@ -12,4 +12,8 @@ JWT_ISS=""
 
 # Resend
 RESEND_API_KEY=""
-RESEND_FROM_EMAIL=""
\ No newline at end of file
+RESEND_FROM_EMAIL=""
+
+# Swagger
+SWAGGER_USER=""
+SWAGGER_PASSWORD=""
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)